src/com/types.ts

```diff
@@ -28,4 +28,27 @@ export type COMModel = COMBaseEntity & {
     loaded: boolean;
 };
 
-export type COMEntity = COMPov | COMLight | COMModel;
+export type COMGeometry = {
+    name: string
+    width: number;
+    height: number;
+    depth: number;
```

Comment:
@ffrank913
Just a small question, why do we need to define a width, height and depth for a Primitive Geometry.
Isn't that the same as just using 1, 1, 1 and scaling it via the scale vector, or am i getting this wrong?

Reply:
The definition for the geometry gives you information about the initial size of a given shape (defined by `name`and interpreted by DIVE in a later step). We need to have some kind of definition for the base geometry.

Counter-question (what is Gegenfrage in english? kekw): How would you tell me what the radius of a cylinder is by only having a scale value?